lib/modules/shipments_api_retriever.rb

```diff
@@ -1,4 +1,5 @@
 module ShipmentsApiRetriever
+  include HTTParty
```

Comment:
I think this can be removed, right? We were using HTTParty before without specifically including the library here.

Reply:
Yes sure, thanks